x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/expression/function/aggregate/Values.java

```diff
@@ -72,9 +72,13 @@ public class Values extends AggregateFunction implements ToAggregator {
             "long",
             "version" },
         preview = true,
-        description = "Returns all values in a group as a multivalued field. The order of the returned values isn’t guaranteed. "
-            + "If you need the values returned in order use <<esql-mv_sort>>.",
+        description = """
+            Returns unique values as a multivalued field. The order of the returned values isn’t guaranteed.
+            If you need the values returned in order use <<esql-mv_sort>>.""",
         appendix = """
+            ::::{note}
+            Use <<esql-top>> if you need to keep repeated values.
```

Comment:
```suggestion
            If you need the values returned in order use [`MV_SORT`](/reference/query-languages/esql/functions-operators/mv-functions.md#esql-mv_sort).""",
        appendix = """
            ::::{note}
            Use [`TOP`](/reference/query-languages/esql/functions-operators/aggregation-functions.md#esql-top) if you need to keep repeated values.
```

nit: if this is for 9.x only we should probably eventually get into the habit of using docsv3 rather than asciidoc syntax 

wdyt @craigtaverner?